docs/index.md

```diff
@@ -91,6 +91,72 @@ resource "grafana_folder" "my_folder" {
 }
 ```
 
+### Installing Synthetic Monitoring on a new Grafana Cloud Stack
+
+```terraform
+// Step 1: Create a stack
+provider "grafana" {
+  alias         = "cloud"
+  cloud_api_key = "<my-api-key>"
+}
+
+resource "grafana_cloud_stack" "sm_stack" {
+  provider = grafana.cloud
+
+  name        = "<StackName>"
+  slug        = "<stackslug>"
```

Comment:
Nit:
```suggestion
  slug        = "<StackSlug>"
```

Reply:
I was trying to show the format of each element but I guess that that wasn't clear 😆 . I'll fix it